evm/src/helpers.ts

```diff
@@ -416,7 +420,7 @@ export const increaseTime5Minutes = async () => {
 }
 
 export const sendToEvm = async (evmMethod: string, ...params: any) => {
-  await web3.currentProvider.sendAsync(
+  await web3.currentProvider.send(
```

Comment:
@se3000 I noticed that sendAsync is bound to send anyway within the top of this file, so this should be alright but i could use your eyes here

Reply:
It would be nice to get rid of that binding. I forget why we do it, but it's always seemed like a hack.

Reply:
it'll get removed when we migrate to ethers anyway